documents/repository.go

```diff
@@ -9,28 +9,6 @@ import (
 	"github.com/syndtr/goleveldb/leveldb"
 )
 
-// LegacyRepository should be implemented by any type that wants to store a document in key-value storage.
-// Deprecated: Use the single collection DB.
-type LegacyRepository interface {
```

Comment:
This PR shouldn't handle this. There is a different PR for that

Reply:
Its just being moved.